src/store/Businesses.js

```diff
@@ -1,29 +1,23 @@
-import { observable, action, decorate } from 'mobx';
+import { observable, action } from 'mobx';
 import api from '../utils/apis';
 
+
 class Businesses {
-    nearby = { data: [], loading: true };
-    currentlyViewing = {};
+    @observable nearby = { data: [], loading: true };
+    @observable currentlyViewing = {};
 
-    fetchNearby = async () => {
+    @action async fetchNearby() {
         const response = await api.searchNearby();
```

Comment:
You should create an ApiStore and set the store in the rootstore so you'll have access to it anywhere in the app